### ISSUE TITLE
Add a setting for blocking Disqus comments

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#5059]
Disqus is a [tracking company](https://source.ind.ie/better/content/tree/master/trackers/disqus.com#notes). It would be nice to be able to block them easily if one doesn't care about reading comments on websites.

While they are [listed on the Disconnect list](https://github.com/disconnectme/disconnect-tracking-protection/blob/a5e14ef3cf674a33ed90059ea5b419f15c98bd35/services.json#L7501-L7506), they are in the "Content" category which we don't use since it has a big impact on webcompat.

**Comment by tomlowenthal** [comment]
This seems like it should be another control along with the Facebook/YouTube/Twitter/LinkedIn list?

**Comment by fmarier** [comment]
Yes, another control in `chrome://settings/socialBlocking` along with YouTube too (https://github.com/brave/brave-browser/issues/4986).

**Comment by fmarier** [comment]
I added Disqus widgets to https://fmarier.github.io/brave-testing/social-widgets.html.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
